File: Hjemmesentral/app/src/main/java/com/homecentral/jrs/hjemmesentral/model/yr/forecast/Symbol.java
package com.homecentral.jrs.hjemmesentral.model.yr.forecast;

import org.simpleframework.xml.Attribute;
import org.simpleframework.xml.Root;

@Root(name = "symbol", strict = false)
public class Symbol {

    @Attribute(name="name", required = false)
    String name;

    @Attribute(name="var", required = false)
    String var;

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public String getVar() {
        return var;
    }

    public void setVar(String var) {
        this.var = var;
    }
}

File: Hjemmesentral/app/src/main/java/com/homecentral/jrs/hjemmesentral/model/yr/forecast/Symbol.kt
package com.homecentral.jrs.hjemmesentral.model.yr.forecast

import org.simpleframework.xml.Attribute
import org.simpleframework.xml.Root

@Root(name = "symbol", strict = false)
data class Symbol @JvmOverloads constructor(
    @field:Attribute(name = "name") var name: String = "",
    @field:Attribute(name = "var") var `var`: String = ""
)
